src/pages/PrivateRoute/index.js

```diff
@@ -10,7 +10,7 @@ class PrivateRoute extends Component {
   render() {
     const { children, auth, dispatch } = this.props;
 
-    if (auth.auth.username === 'admin') {
+    if (auth.auth.username) {
```

Comment:
Is `auth.auth.username` really right here? Where did we get two `auths` deep in state? I see other components with the same `@connect` pattern (`auth: auth.auth`) that use `auth.username`. What's different here? Or is one or the others wrong?

Reply:
Right, nice catch! Just need to destructure the prop before referencing.

Reply:
@dbutenhof should be gone now!